src/components/onboarding/steps/collector_intent/__stories__/collector_intent_page.story.tsx

```diff
@@ -0,0 +1,18 @@
+import { storiesOf } from "@storybook/react"
+import * as React from "react"
+
+import { ContextProvider } from "../../../../artsy"
+import CollectorIntent from "../index"
+
+storiesOf("Onboarding", module).add("Collector Intent", () => {
+  const currentUser = {
+    accessToken: "blah",
+    id: "my_id",
+    name: "Joe",
+  }
+  return (
+    <ContextProvider currentUser={currentUser}>
+      <CollectorIntent />
```

Comment:
@xtina-starr This part updates the story to use the `CollectorIntent` component so we should add that back to master.